### PR TITLE
kiss-chroot: Mount /dev/shm, /dev/pts, /tmp and /run

### LINKED ISSUE #8: kiss-chroot: mount some more paths
It would be nice to have a few extra paths mounted by `kiss-chroot`:
* `/dev/shm`: `python` multiprocessing breaks, which in turn breaks building `firefox` and probably `chromium` too which can be pretty annoying (requires a full rebuild of `python`, maybe a check should be added to the build file aswell ?).
* `/tmp`, `/run`: not really necessary, but helps keep the directories clean.

### PR DESCRIPTION
**Pull request by dylanaraps**
Closes #8

### CODE REVIEW

File: contrib/kiss-chroot [hunk 1]
```diff
@@ -35,20 +38,31 @@ mounted() {
         [ "$target" = "$1" ] && return 0
     done < /proc/mounts
 
+    printf 'mounting %s\n' "$1" >&2
     return 1
 }
 
+set -- "${1%"${1##*[!/]}"}"
```
Comment (dylanaraps) [inline contrib/kiss-chroot:45]: Unrelated but fixes `path/to/chroot//root_fs`.

**Comment by dylanaraps** [comment]
On another note. I've been using my own fork of bubblewrap to replace `kiss-chroot`. It's entirely rootless and correctly mounts all host stuff, etc. Something to think about for later.

**Comment by aabacchus** [comment]
How about also (bind-) mounting `/dev/pts` so that `tty` works, which lets pinentry do its thing for GPG2 password entry?

**Comment by dylanaraps** [comment]
Done.

**Pull request merged**